packages/teleport/src/Nodes/AddNode/Manually/Manually.tsx

```diff
@@ -80,6 +81,14 @@ export default function Manually({
   );
 }
 
+const configDir = '$HOME/.config';
+const configFile = `$HOME/.config/teleport.yaml`;
```

Comment:
Can you abstract these out into a shared location somewhere? I think we should probably set an app level default config output location.

I'm also wondering if we shouldn't use generated config file names as if they run this configure multiple times for apps, nodes, etc it'll override this config.

Reply:
Hey, great ideas, I've added them in this commit https://github.com/gravitational/webapps/pull/751/commits/461a3f64a5eb1e0739583629bd44e227c5161243